cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -2283,16 +2074,22 @@ subroutine strain_rates_U2 (nx_block,   ny_block,  &
  ! MIGHT NOT NEED TO mult by uvm...if done before in calc of uvelU...
       
             ! divergence  =  e_11 + e_22
-            divU    (i,j) = dyU(i,j) * ( uNip1j - uNij ) &
-                          + uvelU(i,j) * uvm(i,j) * ( dyN(i+1,j) - dyN(i,j) ) &
-                          + dxU(i,j) * ( vEijp1 - vEij ) &
-                          + vvelU(i,j) * uvm(i,j) * ( dxE(i,j+1) - dxE(i,j) )
+            ldivU    = dyU(i,j) * ( uNip1j - uNij ) &
+                     + uvelU(i,j) * uvm(i,j) * ( dyN(i+1,j) - dyN(i,j) ) &
```

Comment:
Maybe this is fine, but the ldivT, etc above are within an if(present) block. Shouldn't that be consistent here?